bindata/network/ovn-kubernetes/common/002-rbac-node.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
-  name: openshift-ovn-kubernetes-node
+  name: openshift-ovn-kubernetes-node-ic
```

Comment:
We can go with `openshift-ovn-kubernetes-node-limited`, you are correct, this is not IC specific and the name suggested conveys that we have limited the permissions significantly.

Reply:
I think we will need to iterate further here os it would be good if we have a plan

Would `openshift-ovn-kubernetes-node-limited-v2`, `openshift-ovn-kubernetes-node-limited-v3` work as a scheme going forward? WDYT?

Reply:
I hope we won't need to use the name change thing often. In general this is not really needed if we are expanding the permissions or are removing the ones not used in n-1 release.